fission/src/mirabuf/MirabufLoader.ts

```diff
@@ -353,7 +401,7 @@ class MirabufCachingService {
 
 export enum MiraType {
     ROBOT = 1,
-    FIELD = 2,
+    FIELD,
```

Comment:
Looks like we got safari working! One minor note, was this supposed to be changed?

```suggestion
    FIELD = 2,
```

Reply:
Yep! I just learned the 2 was unnecessary because it increments based on the last one.

Reply:
If it works it works.